packages/nimble-blazor/NimbleBlazor/Components/NimbleRadioButton.razor.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using Microsoft.AspNetCore.Components;
+
+namespace NimbleBlazor;
+
+public partial class NimbleRadioButton : ComponentBase
+{
+    [Parameter]
```

Comment:
I think we're now trying to get in the habit of documenting the public APIs.

Reply:
Added docs